src/main/java/org/medicmobile/webapp/mobile/XWalkMigration.java

```diff
@@ -103,12 +105,14 @@ public void run() {
 		}
 	}
 
+	@SuppressFBWarnings("RV_RETURN_VALUE_IGNORED")
```

Comment:
I am a little concerned about just suppressing this warning (both here and in the methods below).  The doc for the `File.renameTo` method says:

> The return value should always be checked to make sure that the rename operation was successful.
> ...
> Returns: true if and only if the renaming succeeded; false otherwise

This sounds to me like the renaming can actually fail without causing an exception.  It seems worth considering here if we should actually be checking the return value and throwing an error here if we could not rename the file.  What do you think?

I guess it comes down to the question of whether we should continue starting the app if the xWalk migration fails. (But, I am not confident that I know enough about the workflow to make a solid recommendation here...)

Reply:
Same I'm not sure 🤔 maybe the original intention is to ignore the outcome because not significant and just some phones needs to do the migration. I'm going to log a message when it couldn't rename or delete the files just in case.

Reply:
I don't think it should stop the start of the app, since it moves the web resources like local storage, so even if it fails to move these resources from XWalk to WebView, the user should be able to use the app in WebView, just that they might need to download the resource and cache again.

Reply:
Warnings added.